src/models/db/account.js

```diff
@@ -109,7 +116,13 @@ PersistentModelMixin(Account, sequelize, {
   password: Sequelize.STRING,
   public_key: Sequelize.TEXT,
   is_admin: Sequelize.BOOLEAN,
-  is_disabled: Sequelize.BOOLEAN
-})
+  is_disabled: Sequelize.BOOLEAN,
+  fingerprint: Sequelize.STRING
```

Comment:
Added the [schema](https://github.com/interledger/five-bells-shared/pull/50) and migration to this PR